frontend/.vscode/tasks.json

```
@@ -84,6 +84,34 @@
         "cwd": "${workspaceFolder}"
       }
     },
+    {
+      "label": "AF: Clean + Rebuild All (Android)",
```

I think it's easier to see if you're running the right task, if the label include critical information at the beginning.
```suggestion
      "label": "AF-Android: Clean + Rebuild All",
```